### PR TITLE
Add form fields to points

### PR DESCRIPTION
**Pull request by lpassamano**
Prior to this, each point only had a caption. Now you can add in more details about a point on a tour!

### CODE REVIEW

File: app/controllers/points_controller.rb [hunk 1]
```diff
@@ -11,14 +11,20 @@ def create
   end
 
   def index
-    @points = Tour.find(params[:tour_id]).points
+    tour = Tour.find(params[:tour_id])
+    @points = tour.points.includes(:image_blob).includes(:image_attachment)
```
Comment (kliph) [inline app/controllers/points_controller.rb]: Unnecessary chained `.includes` it takes `*args` or "splat args" so `.includes(:image_blob, :image_attachment)` should be fine

File: app/javascript/packs/application/components/Point.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }
+}
+
+Point.propTypes = {
+  point: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    caption: PropTypes.string,
+    description: PropTypes.string,
+    location: PropTypes.string,
+    directins: PropTypes.string,
```
Comment (kliph) [inline app/javascript/packs/application/components/Point.js]: Is this a typo?

File: app/javascript/packs/application/components/PointsList.js [hunk 3]
```diff
@@ -20,7 +21,7 @@ export class PointsList extends Component {
     }
 
     return (
-      <div>{points.map(point => <h4 key={point.id}>{point.caption}</h4>)}</div>
+      <div>{points.map(point => <Point point={point} key={point.id} />)}</div>
```
Comment (kliph) [inline app/javascript/packs/application/components/PointsList.js]: When using ids for keys, I usually like to put a string on the front to decrease the possibility for React throwing a warning about needing unique keys (this [page](https://reactjs.org/docs/lists-and-keys.html#basic-list-component) talks about that warning).  Probably not necessary if you haven't seen the warning, but something that I think about when setting `key` props in React.

File: app/javascript/packs/application/components/Point.js (hunk 2 above)
Comment (lpassamano) [reply app/javascript/packs/application/components/Point.js]: yup! thanks

**Pull request merged**